t/43-dbtest.t

```diff
@@ -5,7 +5,7 @@ if (!defined $ENV{LSMB_TEST_DB}){
         plan skip_all => 'Skipping all.  Told not to test db.';
 }
 else {
-        plan tests => 470;
+        plan tests => 465;
```

Comment:
Using random form_id key instead of sequential. (9a5d554) dropped the amount of tests because the form id is now generated automatically by the database. I think that session ids should use the same method.